### ISSUE TITLE
Node label for path to use

### ISSUE THREAD

**Issue by michaelandrepearce** [rancher/local-path-provisioner#158]
Is it possible rather where path to use is not the default one, from what i understand you have to define this then in the config and having to keep the config in sync with nodes. Is it possible to use a label on the node to define if it uses the default or a custom config on that node, similar in nature to longhorn?

**Comment by yasker** [comment]
Yeah, it's possible. Local path provisioner will need to have read permission for the nodes to get the annotations from nodes and have a setting in the config map to enable it. It will need some design and effort to make it work. PRs are welcome!
